google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionImpl.java

```diff
@@ -208,6 +209,7 @@ static UnitOfWorkType of(TransactionMode transactionMode) {
   private AutocommitDmlMode autocommitDmlMode = AutocommitDmlMode.TRANSACTIONAL;
   private TimestampBound readOnlyStaleness = TimestampBound.strong();
   private QueryOptions queryOptions = QueryOptions.getDefaultInstance();
+  private RpcPriority rpcPriority = null;
```

Comment:
Should it be `private RpcPriority rpcPriority = Priority.PRIORITY_UNSPECIFIED`?

Reply:
Cannot assign value of type Priority to RpcPriority, although I though of adding UNSPECIFIED in RpcPriority enum but after @olavloite suggestions I decided to not add the value https://github.com/googleapis/java-spanner/pull/1548#discussion_r748503248